app/workflow_executors/step_executors/impl/open_ai_next_js_update_code_file_executor.go

```diff
@@ -223,15 +224,14 @@ func (e *NextJsUpdateCodeFileExecutor) InsertCode(filePath string, lineNumber in
 }
 
 func (e NextJsUpdateCodeFileExecutor) UpdateCodeFile(llmResponse, fileName string, step steps.UpdateCodeFileStep) error {
+	fmt.Printf("____updating file %s_____", fileName)
```

Comment:
move to zap logger
also remove unnecessary logs like  `fmt.Println("__payload_____12july", createJobRequest)
`